docs/docsite/rst/playbooks_vault.rst

```diff
@@ -5,7 +5,7 @@ Using Vault in playbooks
 
 New in Ansible 1.5, "Vault" is a feature of ansible that allows keeping sensitive data such as passwords or keys in encrypted files, rather than as plaintext in your playbooks or roles. These vault files can then be distributed or placed in source control.
 
-To enable this feature, a command line tool, `ansible-vault` is used to edit files, and a command line flag `--ask-vault-pass` or `--vault-password-file` is used. Alternately, you may specify the location of a password file or command Ansible to always prompt for the password in your ansible.cfg file. These options require no command line flag usage.
+To enable this feature, a command line tool, ``ansible-vault`` is used to edit files, and a command line flag ``--ask-vault-pass`` or ``--vault-password-file`` is used. Alternately, you may specify the location of a password file or command Ansible to always prompt for the password in your ``ansible.cfg file``. These options require no command line flag usage.
```

Comment:
Could these use the ':option:' link? That shoud point the link at the cli command docs page.
(And a ':ref:' for 'ansible-vault'...)

rst/vault.rst uses that style for ex.

``` rst
:option:`--ask-vault-pass <ansible-vault --ask-vault-pass>` and 

To enable this feature, a command line tool, :ref:`ansible-vault` is used to edit files, and a command line flag 
:option:`--ask-vault-pass <ansible-vault --ask-vault-pass>`  or :option:`--vault-password-file <ansible-vault --vault-password-file>` is used.

```

Reply:
Good idea, done

Reply:
Done